ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -52,13 +53,13 @@ body {
 
 a {
   text-decoration: none;
+
   &:hover {
-    text-decoration: underline;
+    text-decoration: none;
   }
 
   &:focus {
-    text-decoration: underline;
-    background: $kuiFocusBackgroundColor;
+    text-decoration: none;
```

Comment:
These changes broke the focus state of a bunch of stuff (see breadcrumbs). These should be left it because they are a completely common pattern and I'd rather not have to write focus states and hover states for everything.